backend/services/syson-tree-services/src/main/java/org/eclipse/syson/tree/explorer/services/ComposedSysONExplorerService.java

```diff
@@ -143,29 +143,35 @@ public List<Object> getElements(IEditingContext editingContext, List<String> act
                 .orElseGet(() -> this.defaultExplorerService.getElements(editingContext, activeFilterIds));
     }
 
+    @Override
     public String getType(Object self) {
-        StringBuilder type = new StringBuilder();
-        if (self instanceof ViewUsage viewUsage) {
-            var viewDefinition = viewUsage.getViewDefinition();
-            if (viewDefinition != null) {
-                type.append(" [");
-                type.append(viewDefinition.getDeclaredName());
-                type.append("]");
-            }
-        }
-        return type.toString();
+        return this.getDelegate(self)
+                .map(delegate -> delegate.getType(self))
+                .orElseGet(() -> this.defaultExplorerService.getType(self));
     }
 
+    @Override
     public String getShortName(Object self) {
```

Comment:
@AxelRICHARD as we discussed I used this PR to move these methods to the appropriate class and update the corresponding interfaces